xml/docu_styleguide.terminology.xml

```diff
@@ -20,6 +20,21 @@
    <quote>Accepted</quote> in the table below. All terms are reproduced in
    sentence-style capitalization.
   </para>
+   
+   <important>
+     <para>
+       SUSE is a part of the Inclusive Naming Initiative that aims to help authors
+       make consistent, responsible choices to remove harmful language. When you are
+       making language choices for documentation, ensure you review the 
+       <link xlink:href="https://inclusivenaming.org/language/word-list/"
+             <citetitle>Word List</citetitle></link> and the
```

Comment:
Looks like there is a missing ">":

```suggestion
       <link xlink:href="https://inclusivenaming.org/language/word-list/"
             ><citetitle>Word List</citetitle></link> and the
```

Apart from this, if you are using DocBook5 (instead of GeekoDoc), you could shorten it:

```suggestion
       <citetitle xlink:href="https://inclusivenaming.org/language/word-list/"
          >Word List</citetitle> and the
```